libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -470,54 +366,61 @@ - (NSString*)uniqueUserAccountIdentifier {
     return identifier;
 }
 
+-(SFOAuthCredentials *)oauthCredentials {
```

Comment:
It's not clear to me why this is owned on `SFUserAccountManager`, as opposed to being owned—if at all—as part of `SFAuthenticationManager`.